expression/builtin_time.go

```diff
@@ -5088,21 +5088,22 @@ func (b *builtinConvertTzSig) Clone() builtinFunc {
 }
 
 // evalTime evals CONVERT_TZ(dt,from_tz,to_tz).
+// `CONVERT_TZ` function returns NULL if the arguments are invalid.
 // See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_convert-tz
 func (b *builtinConvertTzSig) evalTime(row chunk.Row) (types.Time, bool, error) {
 	dt, isNull, err := b.args[0].EvalTime(b.ctx, row)
 	if isNull || err != nil {
-		return types.Time{}, true, err
+		return types.Time{}, true, nil
```

Comment:
I have no idea whether directly ignore the error is ok. For safety, shall we only check the error is "unknown time zone"?

Reply:
https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_convert-tz
> This function returns NULL if the arguments are invalid.

Other invalid arguments should also be ignored。